### PR TITLE
Add build_break_rules keys

### PR DESCRIPTION
**Pull request by gitnachogo**

### CODE REVIEW

File: docs/integrations/tips.md [hunk 1]
```diff
@@ -57,6 +57,62 @@ SAST Scan Summary
...
+| audit-jsp | JSP Security Audit |
+| source-apex | Apex Source Analyzer |
+| binary | Binary byte-code Analyzer |
+| class | Class File Analyzer |
+| jar | Jar File Analyzer |
+| cpg | ShiftLeft NextGen Analyzer |
+| inspect | ShiftLeft NextGen Analyzer |
+| ng-sast | ShiftLeft NextGen Analyzer |
+| source-ruby | Ruby Source Analyzer |
+| empty-scan | Empty Scan Ignore |
```
Comment (prabhu) [inline docs/integrations/tips.md:114]: empty-scan is a work around for GitHub Code scanning. We can remove it from docs.

File: docs/integrations/tips.md [hunk 2]
```diff
@@ -57,6 +57,62 @@ SAST Scan Summary
...
+| source-java | Java Source Analyzer |
+| source-python | Python Source Analyzer |
+| source-php | PHP Source Analyzer |
+| phpstan | PHP Source Analyzer |
+| audit-python | Python Security Audit |
+| audit-php | PHP Security Audit |
+| taint-php | PHP Security Analysis |
+| taint-python | Python Security Analysis |
+| psalm | PHP Security Audit |
+| /opt/phpsast/vendor/bin/psalm | PHP Security Analysi |
```
Comment (prabhu) [inline docs/integrations/tips.md:97]: Typo `Analysis`

File: docs/integrations/tips.md [hunk 3]
```diff
@@ -57,6 +57,62 @@ SAST Scan Summary
...
+| /opt/pmd-bin/bin/run.sh | Source Code Analyzer |
+| gitleaks | Secrets Audit |
+| gosec | Go Security Audit |
+| tfsec | Terraform Static Analysis |
+| lint-tf | Terraform Static Analysis |
+| shellcheck | Shell Script Analysis |
+| bandit | Security Audit for Python |
+| checkov | Security Audit for Infrastructure |
+| source-aws | Security Audit for AWS |
+| source-arm | Security Audit for Azure Resource Manag |
```
Comment (prabhu) [inline docs/integrations/tips.md:77]: Typo `Manager`

File: docs/integrations/tips.md (hunk 1 above)
Comment (gitnachogo) [reply docs/integrations/tips.md:114]: sure! I just used all the keys available in the config.py, but I'll remove it

**Review by prabhu — APPROVED** [review]
Thank you very much!

**Pull request merged**